hangman.js

```diff
@@ -1 +1,107 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word = prompt('gimme a word:');
+var guesses = [];
+var attempts = word.length + 4;
+var attemptsLeft = attempts;
+var display = '';
+
+myl = 'a';
+place = word.indexOf(myl);
+
+function showDisplay() {
+  for(i = 0; i < word.length; i++) {
+    display += '-';
+  }
+  display;
```

Comment:
I think you meant to write `return display;` instead of just referencing the variable here. However, since you are using `var display` as a global variable, and you are not passing any arguments into your functions, you don't really need to return it in this case.